draft-ietf-quic-v2.md

```diff
@@ -208,12 +208,33 @@ are willing to suffer a round-trip penalty if they are incorrect.
 # Applicability
 
 This version of QUIC provides no change from QUIC version 1 relating to the
-capabilities available to applications. Therefore, all Application Layer
+capabilities available to applications. Therefore, most Application Layer
 Protocol Negotiation (ALPN) ({{?RFC7301}}) codepoints specified to operate over
-QUICv1 can also operate over this version of QUIC.
+QUICv1 can also operate over this version of QUIC. Applications SHOULD NOT
+require new ALPNs for QUICv2 unless there is a specific performance or
+interoperability concern.
 
 All QUIC extensions defined to work with version 1 also work with version 2.
 
+## The HTTP/3 Exception
+
+HTTP/3 can use the ALPN codepoint to discover QUIC versions via the Alt-Svc
+HTTP field {{?RFC7838}}. If different QUIC versions use the same HTTP/3 ALPN, a
+client might discover QUIC support via Alt-Svc on a TCP connection, and then
+try QUIC only to find the server is incompatible. Therefore, HTTP/3 is an
+exception to the principle that new QUIC versions do not require new ALPN
+codepoints.  
+
+Therefore, HTTP/3 servers using QUIC version 2 MUST accept the "h3v2" ALPN for
```

Comment:
I would prefer this bikeshed to be painted in "h3q2" color, because "h3v2" sounds like a revision of HTTP/3 to me